ietf/templates/doc/mail/ballot_writeup.txt

```diff
@@ -26,12 +27,14 @@ Document Quality
    Review, on what date was the request posted?
 
 Personnel
-
+{% if doc.shepherd and doc.ad %}{% filter wordwrap:"76" %}
+   The Document Shepherd for this document is {{ doc.shepherd.person.name }}. The Responsible Area Director is {{ doc.ad.name }}.{% endfilter %}
+{% else %}
```

Comment:
This would cause the reminder to the AD or shepherd to track the assignment of the IANA experts to be suppressed - is that intentional?

Reply:
I can't recall when an AD would have left that in/filled that out. Maybe that should be removed entirely.

Reply:
wwfm

Reply:
I'll remove it.

Should I make the "IRTF Note" part conditional on the doc being on that stream? And ditto for the "IESG Note"?

And what is the "IANA Note" for?

Reply:
Looks like IANA Note is frequently used, even now. See https://mailarchive.ietf.org/arch/browse/ietf-announce/?q=IANA%20Note for examples.